data/src/RobotParameters.cpp

```diff
@@ -0,0 +1,46 @@
+//
+// Created by rolf on 26-10-20.
+//
+
+
+#include "RobotParameters.h"
+
+RobotParameters RobotParameters::DEFAULT() {
+    RobotParameters parameters; //default constructor should be roughly correct
+    return parameters;
+}
+
+RobotParameters RobotParameters::RTT_2020() {
+    //TODO: fix these
+    RobotParameters parameters(0.09,0.15,0.1,0.1,0.0);
+    return parameters;
+}
+
+proto::RobotParameters RobotParameters::toProto() const {
+    proto::RobotParameters protoMsg;
+    protoMsg.set_radius(radius);
+    protoMsg.set_height(height);
+    protoMsg.set_front_width(frontWidth);
+    protoMsg.set_dribbler_width(dribblerWidth);
+    protoMsg.set_angle_offset(angleOffset);
+    return protoMsg;
+}
+
+RobotParameters::RobotParameters(const proto::RobotParameters &protoParams) :
+        radius{protoParams.radius()},
+        height{protoParams.height()},
+        frontWidth{protoParams.front_width()},
+        dribblerWidth{protoParams.dribbler_width()},
+        angleOffset{protoParams.angle_offset()} {
+}
+
+//TODO fix correct values
+RobotParameters::RobotParameters() : radius{0.09}, height{0.15},frontWidth{0.1}, dribblerWidth{0.1}, angleOffset{0.0} {
```

Comment:
No magic numbers pls :)

Reply:
By their very definition these are magic numbers, because they define an 'average' SSL robot.

Reply:
So I don't really know what I can do here.

Reply:
Maybe put them in constants.cpp? Not sure.